swagger.json

```diff
@@ -2019,72 +2032,577 @@
               "$ref" : "#/definitions/Error"
             }
           },
-          "403" : {
-            "description" : "Forbidden",
```

Comment:
As a general coment: This go swagger tool generates client code that does check the response code. If a response code that has not been listed here is encountered, then the generated go client calls "panic". See: https://github.com/vmware/vcf-sdk-go/blob/b799f0bb0eb9fdf7430b9467e18faf5da0dd86d1/client/certificates/certificates_client.go#L514

This leads the provider to majorly fail (not just one resource, whoose API is being called) with some very uninformative message. 

Return codes, thus, have a major impact on UX